test/query.js

```diff
@@ -902,14 +933,26 @@ describe('Queries', function () {
 
     it('should perform a background query that executes the operations #slow', async function () {
       const query = client.query(helper.namespace, testSet)
-      const ops = [op.incr('backgroundOps', 1)]
+      const ops = [op.write('backgroundOps', 1)]
```

Comment:
Why?

Reply:
The same bin is incremented in other tests.  Writing here instead of incrementing allows the result in the bin to be a consistent value rather than dependent on the execution of other tests.

Here is the test failure for more information:

`  1) Queries
       query.operate()
         should perform a background query that executes the operations #slow:

      AssertionError: expected 3 to equal 1
      + expected - actual

      -3
      +1
      
      at Context.<anonymous> (test/query.js:942:44)

`